classificators/src/main/java/ru/okpdmarket/controller/ClassificatorController.java

```diff
@@ -50,7 +51,10 @@ public ClassificatorItem getItem(@PathVariable(value = "classificatorCode") Stri
     }
 
     @RequestMapping(value = "/{code}/search", method = RequestMethod.GET)
-    public List<ClassificatorItem> search(@PathVariable(value = "code") String classificatorCode, @RequestParam String query) {
-        return searchService.searchByClassificator(classificatorCode, query);
+    public SearchResult search(@PathVariable(value = "code") String classificatorCode,
+                               @RequestParam(value = "query") String query,
+                               @RequestParam(value = "take", defaultValue = "${application.search.defaultTake}") Integer take,
+                               @RequestParam(value = "offset", defaultValue = "0") Integer offset) {
```

Comment:
Shouldn't this default value also be in properties?

Reply:
Nope, what for? Can you imagine any usecase?

Reply:
ok, I can't